docs/internal.md

```diff
@@ -64,6 +67,46 @@ yarn package
 ```
 
 This will generate a file `dist/web-ext-artifacts/sporran-???.zip` ready to be uploaded to stores.
+This version will only connect to _KILT Spiritnet_ blockchain via predefined node endpoints.
+
+### For the internal version
+
+Run the following commands:
+
+```
+yarn install
+yarn build-internal
+yarn package
+```
+
+This will generate a file `dist/web-ext-artifacts/sporran-???.zip` ready for developers to use.
+This version will connect to any node endpoints the user inputs and by default to the _KILT Peregrine_ blockchain.
+
+## Uploading to the release's site
+
+We recommend external developers to download the test version of the Sporran extension from [the releases' page on GitHub](https://github.com/BTE-Trusted-Entity/sporran-extension/releases).
+The uploads to this page occur manually.
+Only Sporran versions using changes merged to `main` should be published as releases.
+
+To get a releasable Sporran Extension file you could either [build and package it locally](#For-the-internal-version) or, alternatively, follow these steps:
+
+1. Merge the wished changes to `main`.
+2. Go to the [Sporran GitHub's Actions page for "Packages the internal version"](https://github.com/BTE-Trusted-Entity/sporran-extension/actions/workflows/publish.yaml).
+3. Click on the workflow of the wished commit.
+4. Download the artifact.
+
+Before updating it to the release site, please prepend a _"TEST-"_ to the zip file's name.
+If you got the extension from the GitHub workflow, it would be nice to replace the commit hash with the sporran version on the zip's name.
```

Comment:
```suggestion
If you got the extension from the GitHub workflow, it would be nice to replace the commit hash with the Sporran version on the zip's name.
```

Reply:
implemented